src/aac/aacconfig.h

```diff
@@ -97,26 +97,34 @@ class AACSpecificConfig
 		
 		//Put bytes
 		BitWritter writter(data,size);
+
+		try
+		{
 		
-		//object type - AAC-LC
-		writter.Put(5,objectType);
-		//Get rate index
-		BYTE index = GetSampleRateIndex(rate);
-		//Set index rate
-		writter.Put(4, index);
-		//If not found
-		if (index==0x0F)
-			//Set rate
-			writter.Put(24,rate);
-		//Set cannles
-		writter.Put(4, channels);
-		//GASpecificConfig
-		writter.Put(1, frameLength); 
-		writter.Put(1, coreCoder); 
-		writter.Put(1, extension); 
-		
-		//Flush
-		return writter.Flush();
+			//object type - AAC-LC
+			writter.Put(5,objectType);
+			//Get rate index
+			BYTE index = GetSampleRateIndex(rate);
+			//Set index rate
+			writter.Put(4, index);
+			//If not found
+			if (index==0x0F)
+				//Set rate
+				writter.Put(24,rate);
+			//Set cannles
+			writter.Put(4, channels);
+			//GASpecificConfig
+			writter.Put(1, frameLength); 
+			writter.Put(1, coreCoder); 
+			writter.Put(1, extension); 
+
+			//Flush
+			return writter.Flush();
+		}
+		catch (std::exception& e) 
+		{
+			return 0;
```

Comment:
Should we log these exceptions and below? It doesnt look like the callers always log errors? For writes at least it should be rare and not spammy and actually if we are seeing lots of corrupt input and it is spammy then we probably want to know about it.

Someday we will probably allow the exceptions to propagate out and give better info of the cause to the caller, but for now it may be useful to know if it was a logic or runtime error at least.

Reply:
Should not happen ever, but doesn't harm to add a warning.
```suggestion
			return Warning("-AACSpecificConfig:Serialize() | Could not serialize aac config\n");
```